hbase-rest/src/main/java/org/apache/hadoop/hbase/rest/RESTServer.java

```diff
@@ -94,6 +94,8 @@
   static final String REST_CSRF_METHODS_TO_IGNORE_DEFAULT = "GET,OPTIONS,HEAD,TRACE";
   public static final String SKIP_LOGIN_KEY = "hbase.rest.skip.login";
   static final int DEFAULT_HTTP_MAX_HEADER_SIZE = 64 * 1024; // 64k
+  static final String HTTP_HEADER_CACHE_SIZE = "hbase.rest.jetty.httpConfig.headerCacheSize";
```

Comment:
[nit] maybe use non camel case? although it's very long.
```suggestion
  static final String HTTP_HEADER_CACHE_SIZE = "hbase.rest.http.header.cache.size";
```

Reply:
Hm. I was trying to preserve the jetty property name, but you are right that it looks a bit long.